shopelectro/tests/tests_views.py

```diff
@@ -522,3 +525,29 @@ def test_admin_autocomplete_has_no_model_pages(self):
             f'?term={self.QUOTED_SIGNLE_RESULT_TERM}&pageType=category'
         )
         self.assertTrue(len(json_to_dict(response)) == 1)
+
+
+class Redirects(TestCase):
+
+    fixtures = ['dump.json']
+
+    # will be resurrected at rf#140
+    @unittest.expectedFailure
+    def test_redirect_on_existing_page(self):
```

Comment:
Maybe `test_redirect_from_existing_page` will be more relevant. decide by yourself